vllm/compilation/collective_fusion.py

```diff
@@ -1069,6 +1069,7 @@ def replacement(quant_result: torch.Tensor, residual: torch.Tensor,
 
 
 class AllReduceFusionPass(VllmInductorPass):
+    disabled = True
```

Comment:
anywhere we turn on this pass?

Reply:
This happens inside the constructor, not sure why this issue happened